sound/soc/sof/ipc4-topology.h

```diff
@@ -71,15 +72,27 @@
  * @mem_usage: Memory usage
  * @state: Pipeline state
  * @msg: message structure for pipeline
+ * @skip_fe_trigger: skip triggering this pipeline in the FE DAI trigger
```

Comment:
yeah, well. skip_fe_trigger doesn't mean skip during the FE trigger. Confusion between cause and effect I am afraid.